opencraft/settings.py

```diff
@@ -447,6 +453,10 @@
 DEFAULT_INSTANCE_MYSQL_URL = env('DEFAULT_INSTANCE_MYSQL_URL', default=None)
 DEFAULT_INSTANCE_MONGO_URL = env('DEFAULT_INSTANCE_MONGO_URL', default=None)
 
+# The RabbitMQ host must be accessible from both OpenCraft IM as well as well as any instances using it.
+INSTANCE_RABBITMQ_HOST = env('INSTANCE_RABBITMQ_HOST', default=None)
+INSTANCE_RABBITMQ_PORT = env('INSTANCE_RABBITMQ_PORT', default=5671)
```

Comment:
The new variables still need doc - cf https://github.com/open-craft/opencraft/pull/152#discussion_r87029553